src/components/predictive-layer/predictive-layer.vue

```diff
@@ -9,13 +9,17 @@
         v-slot="{ identifierResult }"
```

Comment:
It seems that the query suggestions are not being highlighted.

<img width="426" alt="image" src="https://user-images.githubusercontent.com/75546736/213674711-6b6b621f-326f-4490-84dd-f29f8b163564.png">